### ISSUE TITLE
How to put back accessions in Ingest for DSP submissions not tracked in Ingest? 

### ISSUE THREAD

**Issue by aaclan-ebi** [ebi-ait/hca-ebi-dev-team#226]
The archiver is now tracking archive submissions and archive entities in Ingest database. These database entities are being used in the archiver to store back the accessions. There are some datasets which are already submitted before this change.

How should we put back the accessions for those datasets?

Dev notes:

1. We could create a script populate those entities in the database using the DSP submission url and run the archiver again. It might be tricky to add the "metadataUuids",  "accessionedMetadataUuids" fields.

2. Need to remove that submission uuid uniqueness in archiveSubmissions collection. Ingest submission to DSP submissions can be one-to-many. 

Example: 
1st DSP submission can have project, study, sample, sequencingExperiment.
2nd DSP submission can have sequencingRuns referencing sequencingExperiment from 1st submission.

**Comment by aaclan-ebi** [comment]
The solution must be able to be applied to the 2 datasets linked in this ticket.
1. Evan Macosko - Human brain cell-type variation for GA
2. KidneySingleCellAtlas

**Comment by lauraclarke** [comment]
Is this blocked by the update ticket? https://app.zenhub.com/workspace/o/ebi-ait/hca-ebi-dev-team/issues/201

**Comment by aaclan-ebi** [comment]
If we're going to go with the no. 1 in Dev notes, there'll be no need to create an update submission for this. We'll just rerun the archiver and the archiver will update the metadata with accessions.

**Comment by ESapenaVentura** [comment]
Also I kinda forgot about this dataset as well https://github.com/ebi-ait/hca-ebi-wrangler-central/issues/20

We should make sure this one gets the accessions back into ingest

**Comment by aaclan-ebi** [comment]
Hi @lauraclarke, this ticket still needs some work, why was it moved to Review/QA? 

I'll put it back to Sprint backlog.

**Comment by lauraclarke** [comment]
Thanks @aaclan-ebi I think I might have gotten in confused with another ticket

**Comment by ESapenaVentura** [comment]
Hi @aaclan-ebi this no longer applies to macosko's dataset, as we had to re-submit

**Comment by lauraclarke** [comment]
@aaclan-ebi @ESapenaVentura I think I am missing a final list of projects that need archive accessions updating in them and if there is anything odd about what is needed

Am I correct in thinking there are three and there are all DCP1

https://app.zenhub.com/workspaces/ingest-dev-5cfe1cb26482e537cf35e8d1/issues/ebi-ait/hca-ebi-wrangler-central/18

https://app.zenhub.com/workspaces/ingest-dev-5cfe1cb26482e537cf35e8d1/issues/ebi-ait/hca-ebi-wrangler-central/17

Then the newest submission to the Tissue Sensitivity dataset
https://github.com/ebi-ait/hca-ebi-wrangler-central/issues/20

Is this the complete list?
Is there any complexities about any of these datasets that whoever takes this on needs to be understood?

**Comment by aaclan-ebi** [comment]
Yes so the list of datasets here will be:
1. KidneySingleCellAtlas ebi-ait/hca-ebi-wrangler-central#18
2. Human Hematopoietic Profiling ebi-ait/hca-ebi-wrangler-central#17
3. Tissue Sensitivity ebi-ait/hca-ebi-wrangler-central#issues/20

@lauraclarke i've put some notes on the ticket description above. Feel free to ask any clarifications.

**Comment by lauraclarke** [comment]
@mshadbolt @ESapenaVentura @rays22 

I am wondering if these would be better as update tickets on the wrangling board, should I create a ticket for each dataset where we still may need to add the archive accessions to the existing ingest record and put in updates column? is there any label I should use?

**Comment by mshadbolt** [comment]
I kind of want the wrangling board to be at the granularity of entire datasets, not multiple tickets per dataset, otherwise it will be difficult to use it for tracking purposes

**Comment by lauraclarke** [comment]
Very happy for there to be a ticket per dataset that needs to be updated, I agree that makes this easier to track

Do we resurrect the old tickets for these datasets or create new ones?

**Comment by mshadbolt** [comment]
The way I kind of envisioned it was that each dataset has a single dataset tracker ticket, and if we come back to it later we can reopen the ticket, but not sure if other wranglers agree...

Also it could be that the datasets mentioned on this ticket do not yet have a 'dataset/project tracker' ticket in the new repo since they are kind of old.

**Comment by lauraclarke** [comment]
Yep, that is very possible, how about I look to see if there are existing tickets and reopen them if there are ones and if there aren't ones, I can create new ones specifically for this update

I am guessing there is a data tracking sheet column I should then put the ticket link into? can you tell me what to look for

**Comment by mshadbolt** [comment]
In the tracker the column to put the link is called 'github_link'

**Comment by lauraclarke** [comment]
These tickets have been reopened and the links added to the data tracker spreadsheet and they have been put in the updates column. I will close this ticket
